DiamondLightSource/web-ui-components
# [LIMS-1493] Log scale

**JIRA ticket**: [LIMS-1493](https://jira.diamond.ac.uk/browse/LIMS-1493)

**Summary**:

- Add log scale options for `BoxPlot` and `ScatterPlot`

**Changes**:

- Add log scale options for `BoxPlot` and `ScatterPlot`

**To test**:
- Run storybook, go to http://localhost:6006/?path=/story/components-plots-box--log-axis, ensure Y axis is displayed in log scale
-  Go to http://localhost:6006/?path=/story/components-plots-scatter--with-log-y-axis, ensure Y axis is displayed in log scale
-  Go to http://localhost:6006/?path=/story/components-plots-scatter--with-log-y-axis, ensure X axis is displayed in log scale
